CondFormats/HGCalObjects/interface/HGCalESProducerTools.h

```diff
@@ -0,0 +1,97 @@
+#ifndef CondFormats_HGCalObjects_HGCalESProducerTools_h
+#define CondFormats_HGCalObjects_HGCalESProducerTools_h
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include <string>
+#include <sstream>    // for std::istringstream
+#include <fnmatch.h>  // to match glob patterns
```

Comment:
Elsewhere we tend to use `std::regex` and imitating globbing by replacing `*` in a glob pattern with `.*` for a regex.

Reply:
as in FWCore/Utilities/src/RegexMatch.cc .

Reply:
Done in https://github.com/cms-sw/cmssw/pull/48137/commits/5ba71768546ae894cbffa898b9c8b1bd9cad9a5b.